repos/system_upgrade/el7toel8/actors/checksystemarch/actor.py

```diff
@@ -21,8 +21,10 @@ class CheckSystemArch(Actor):
 
     def process(self):
         if platform.machine() != 'x86_64':
-            report_generic(
-                title='Unsupported arch',
-                summary='Upgrade process is only supported on x86_64 systems.',
-                severity='high',
-                flags=['inhibitor'])
+            create_report([
+                reporting.Title('Unsupported arch'),
```

Comment:
```suggestion
                reporting.Title('Unsupported architecture'),
```

Reply:
Yep, makes sense. We should use this refactoring to improve/fix older reports...